stack/lab/Dockerfile

```diff
@@ -105,3 +105,9 @@ ENV NOTEBOOK_ARGS \
      "--MappingKernelManager.cull_interval=300" \
      "--TerminalManager.cull_inactive_timeout=600" \
      "--TerminalManager.cull_interval=60"
+
+# Set up the logo of notebook interface
+COPY --chown=${NB_UID}:${NB_GID} aiidalab-wide-logo.png ${CONDA_DIR}/notebook-logo.png
+# the folder of logo.png is get by find "$CONDA_DIR/lib" -path "*/notebook/static/base/images/logo.png"
```

Comment:
I am very confused by this code. Why is this `find` necessary? Isn't the original logo always in the same path?

Reply:
> Isn't the original logo always in the same path?

Not exactly the path now is `/opt/conda/lib/python3.9/site-packages/notebook/static/base/images/logo.png`, thus also dependent on the python version.

Reply:
Right, but we control the python version and it's value is available in the Dockerfile you only need to somehow get rid of the patch version.e.g 3.9.13 to 3.9, but I am assuming that should be possible.

Reply:
Did not manage to do it simply. 

I can do:
```
RUN export PYTHON_VERSION_MAJOR_MINOR=$(python -c "import sys; print('python{}.{}'.format(*sys.version_info[:2]))") && \
    mv /tmp/notebook-logo.png "$CONDA_DIR/lib/$PYTHON_VERSION_MAJOR_MINOR/site-packages/notebook/static/base/images/logo.png"
```

But I don't think it is simple. If read from `PYTHON_VERSION` then the problem is it can be `3.9` or `3.9.13`, so it is not easy to parse the `<major>.<minor>` from it.

Reply:
I add a comment to explain the usage of `find`, I assume the `-exec` is a common option to execute the command on the result of `find` command.